src/api/java/jni/java_api.h

```diff
@@ -13,31 +13,40 @@
  * The cvc5 Java API.
  */
 
-#include <jni.h>
-
 #ifndef CVC5__JAVA_API_H
 #define CVC5__JAVA_API_H
 
+#include <jni.h>
```

Comment:
The filename suggests that this file contains the main api, while it only holds a few helpers. Maybe rename to `api_utilities`?

Reply:
Renamed to `api_utilities`